target/linux/armsr/base-files/lib/preinit/79_move_config

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+move_config() {
+	local partdev
+
+	. /lib/upgrade/common.sh
+
+	if export_bootdevice && export_partdevice partdev 1; then
+		if mount -t vfat -o rw,noatime "/dev/$partdev" /mnt; then
```

Comment:
No hardcoding... use part_magic_fat.... actually this can be just a copy of the x86 file...

Reply:
Changed, but the boot partition should always be fat?